metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -160,8 +161,14 @@ def _exec_col_query(self, table_uri: str) -> Tuple:
         OPTIONAL MATCH (col:Column)-[:DESCRIPTION]->(col_dscrpt:Description)
         OPTIONAL MATCH (col:Column)-[:STAT]->(stat:Stat)
         OPTIONAL MATCH (col:Column)-[:HAS_BADGE]->(badge:Badge)
-        RETURN db, clstr, schema, tbl, tbl_dscrpt, col, col_dscrpt, collect(distinct stat) as col_stats,
-        collect(distinct badge) as col_badges
+        OPTIONAL MATCH (col:Column)-[:TYPE_METADATA]->(Type_Metadata)-[:SUBTYPE *0..]->(tm:Type_Metadata)
+        OPTIONAL MATCH (tm:Type_Metadata)-[:DESCRIPTION]->(tm_dscrpt:Description)
+        OPTIONAL MATCH (tm:Type_Metadata)-[:HAS_BADGE]->(tm_badge:Badge)
```

Comment:
My take on badges is that it should be fine to have badges of category column on the type metadata nodes since it will probably serve the same function of indicating something about a field at a glance without being clickable